.circleci/config.yml

```diff
@@ -27,35 +26,40 @@ sbt-build: &sbt-build
           - ~/.m2
           - ~/.ivy2
           - ~/.sbt
-    - store_artifacts:
-        path: core/target/scala-2.12/api/
-        destination: scaladoc/core
-    - store_artifacts:
-        path: core/target/test-reports/
-        destination: test-reports/core
-    - store_test_results:
-        path: core/target/test-reports/
-
 
 jobs:
 
   test:
     <<: *sbt-build
-    resource_class: large
     environment:
       SBT_COMMAND: "; compile; clean; coverage; test; coverageReport; coverageAggregate; doc"
 
-  integration-test-fast:
-    <<: *sbt-build
-    resource_class: large
-    environment:
-      SBT_COMMAND: "it:testOnly -- -l org.scalatest.tags.Slow"
-
-  integration-test-all:
-    <<: *sbt-build
-    resource_class: large
-    environment:
-      SBT_COMMAND: "it:test"
+  integration-test:
+    docker:
+      - image: docker/compose:1.21.2
+    steps:
+      - run: apk update && apk add curl git openssh-client coreutils
+      - setup_remote_docker
+      - checkout
+      - run:
+          name: Update submodules
+          command: |
+            git submodule sync
+            git submodule update --init --remote
+      - run: docker-compose build --no-cache
+      - run:
+          name: Collect logs
```

Comment:
@rootmos maybe factor out to a "function" or whatever is the right terminology in the scope of yaml.

Reply:
Might be good, especially for other people to ~~steal~~borrow it